tidy.Rmd

```diff
@@ -1,3 +1,8 @@
+---
```

Comment:
Can you please remove this?

Reply:
Sure! Fixed in 27dfedf454afbae25a6d24f00c819bb1a9264a41